src/theme/theme.ts

```diff
@@ -54,6 +54,7 @@ export default {
 		label : "#3B424E", //Same as gray700
 		almostBlack : "#1A1A1A",
 		errorBackground: "#B100000D",
+		white: '#FFFFFF'

```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/simpleviewinc/sv-mosaic/pullRequest?prid=9446470)
```suggestion
		white: "#FFFFFF"
```